usr/bin/update-i2pbrowser

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+## Copyright (C) 2012 - 2018 ENCRYPTED SUPPORT LP <adrelanos@riseup.net>
+## See the file COPYING for copying conditions.
+
+if [ ! "$tpo_downloader_debug" = "" ]; then
+   if [ "$tpo_downloader_debug" -ge "1" ]; then
+      set -x
+   fi
+fi
+
+set -o pipefail
+set -o errtrace
+
+SCRIPTNAME="$(basename "$BASH_SOURCE")"
+IDENTIFIER="i2pbrowser-downloader"
+
+ICON="/usr/share/icons/anon-icon-pack/tbupdate.ico"
```

Comment:
This variable and the two above won't be available in `update-torbrowser`. You would have to `export` them. Below the variables:

    export SCRIPTNAME IDENTIFIER ICON

Then also these variables should be set in `update-torbrowser` only if not already set, i.e. change to:

     [ -n "$IDENTIFIER" ] || IDENTIFIER=.......